sdk/eventhub/event-hubs/samples/runsamples.ts

```diff
@@ -0,0 +1,23 @@
+import * as dotenv from "dotenv";
+dotenv.config();
+
+import { main as sendEventsMain } from "./sendEvents";
+import { main as receiveEventsMain } from "./receiveEvents";
+import { main as receiveEventsUsingCheckpointStoreMain } from "./receiveEventsUsingCheckpointStore";
+import { main as useWithIotHubMain } from "./useWithIotHub";
+import { main as usingAadAuthMain } from "./usingAadAuth";
+import { main as websocketsMain } from "./websockets";
+
+async function main() {
```

Comment:
I like this approach and I see why you wrapped the main() invocations in that environment variable now. Maybe we should just standardize around that model and then set up a command in the package itself for running all the samples by setting RUNNING_IN_TESTS and invoking this script.

Reply:
I made it a little wrapper (and also changed the environment variable name to DO_NOT_EXECUTE_SAMPLE) which I thought would also be more clear.